### PR TITLE
fix: [SRM-14466]: Update correct icons and display title text

### PR DESCRIPTION
**Pull request by praneshg239**
#### Screenshots

Pasted Graphic 2
Pasted Graphic 1

#### PR Checklist

<details>
<summary>Please check if your PR fulfils the following requirements</summary>

- [ ] Tests for the changes have been added. Ideally, include a test that fails without this PR but passes with it.
- [ ] Docs have been [added/updated](https://harness.atlassian.net/jira/software/c/projects/DOC/boards/40).
</details>

<details>
<summary>Use the following comments to re-trigger PR Checks</summary>

- Jest: `retrigger jest`
- Prettier: `retrigger prettier`
- Type Check: `retrigger typecheck`
- ESLint: `retrigger eslint`
- Sonar: `retrigger sonar`
- Standards: `retrigger standards`
- Build: `retrigger build`
- Title Check: `retrigger titlecheck`
- Feature Name Check: `trigger featurenamecheck`
- Coverage: `retrigger coverage`
- Rebase: `trigger rebase`
- Cypress Rest: `retrigger cypress-rest`
- Cypress CD: `retrigger cypress-cd`
- Cypress Pipeline: `retrigger cypress-pipeline`
- Cypress CV: `retrigger cypress-cv`
- Fix Prettier: `fix prettier`
</details>

### CODE REVIEW

**Review by karansaraswat19 — CHANGES_REQUESTED** [review]

File: src/modules/85-cv/pages/monitored-service/components/ServiceHealth/components/ChangesAndServiceDependency/components/ChangesTable/components/ChangeEventCard/__tests__/ChangeEventCard.mock.ts [hunk 1]
```diff
@@ -140,3 +140,28 @@ export const HarnessCDMockData = {
...
+    monitoredServiceIdentifier: 'monitored_service_identifier',
+    serviceName: 'service1',
+    envIdentifier: 'env1',
+    environmentName: 'env1',
+    name: 'CD Nextgen - service1 - test',
+    changeSourceIdentifier: 'cdNg',
+    eventTime: 1633853633448,
+    metadata: {},
+    category: 'Deployment' as any,
+    type: 'HarnessFF' as any
```
[the remark this answers is not in the document]
Comment (praneshg239) [reply src/modules/85-cv/pages/monitored-service/components/ServiceHealth/components/ChangesAndServiceDependency/components/ChangesTable/components/ChangeEventCard/__tests__/ChangeEventCard.mock.ts:163]: It internally renders `InternalCSEventCard` component only right?
```ts
      case ChangeSourceTypes.HarnessFF:
      case ChangeSourceTypes.CustomFF:
      case ChangeSourceTypes.CustomDeploy:
      case ChangeSourceTypes.CustomIncident:
      case ChangeSourceTypes.CustomInfrastructure:
      case ChangeSourceTypes.HarnessCE:
        return <InternalCSEventCard data={data?.resource} />
```

[the remark this answers is not in the document]
Comment (deepesh-ui) [reply src/modules/85-cv/pages/monitored-service/components/ServiceHealth/components/ChangesAndServiceDependency/components/ChangesTable/components/ChangeEventCard/__tests__/ChangeEventCard.mock.ts:163]: We depend on type field , so UI wise we should not face any issue
Lets use the already present mock data **HarnessFFMockData** from  InternalCSEventCard.mock.ts

**Review by karansaraswat19 — APPROVED** [review]

**Comment by karansaraswat19** [comment]
retrigger jest

File: src/modules/85-cv/pages/monitored-service/components/ServiceHealth/components/ChangesAndServiceDependency/components/ChangesTable/components/ChangeEventCard/__tests__/ChangeEventCard.mock.ts (hunk 1 above)
[the remark this answers is not in the document]
Comment (praneshg239) [reply src/modules/85-cv/pages/monitored-service/components/ServiceHealth/components/ChangesAndServiceDependency/components/ChangesTable/components/ChangeEventCard/__tests__/ChangeEventCard.mock.ts]: Updated

**Review by deepesh-ui — APPROVED** [review]

**Pull request merged**